### PR TITLE
wrap with conditional

### PR DESCRIPTION
**Pull request by maryam-mv**
Sorry quick fix.

### CODE REVIEW

File: website/dynamic-blocks/headings.js [hunk 1]
```diff
@@ -34,12 +34,13 @@ const Component = ({ heading, size, level, addTableContent, subText, text }) =>
 					</Cell>
 				</Grid>
 			</WestpacHeading>
-
-			<Grid columns={12}>
-				<Cell width={10} left={[1, 1, 2, 2]}>
-					{subText && text && <p css={{ lineHeight: 2 }}>{text}</p>}
-				</Cell>
-			</Grid>
+			{subText && text && (
```
Comment (dominikwilkowski) [inline website/dynamic-blocks/headings.js:37]: what's `subText` and why is it required for the blow below?

Comment (jaortiz) [reply website/dynamic-blocks/headings.js:37]: Kate asked for the subText to be conditionally shown.

Comment (dominikwilkowski) [reply website/dynamic-blocks/headings.js:37]: We we don't display `subText` anywhere below? is that a checkbox somewhere?

Comment (maryam-mv) [reply website/dynamic-blocks/headings.js:37]: Yeah subText is a boolean that comes from a checkbox.

Comment (dominikwilkowski) [reply website/dynamic-blocks/headings.js:37]: Makes sense then :) thank you both

Comment (maryam-mv) [reply website/dynamic-blocks/headings.js:37]: Screen Shot 2020-04-17 at 1 14 13 pm

**Pull request merged**